7.figures/nbconverted/f1_score_visualization.r

```diff
@@ -0,0 +1,72 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+
+# Load figure themes and colors
+source("figure_themes.R")
+
+# Set output files
+output_file <- file.path(
+    "figures", "f1_scores_multiclass.png"
+)
+
+# Load data
+results_dir <- file.path(
+    "..", "3.evaluate_model", "evaluations", "F1_scores"
+)
+results_file <- file.path(results_dir, "compiled_F1_scores.tsv")
+
+f1_score_df <- readr::read_tsv(
+    results_file,
+    col_types = readr::cols(
+        .default = "d",
+        "Phenotypic_Class" = "c",
+        "data_split" = "c",
+        "shuffled" = "c",
+        "feature_type" = "c"
+    )
+) %>%
+    dplyr::select(!`...1`) %>%
+    dplyr::mutate(feature_type_with_data_split = paste0(feature_type, data_split)) %>%
+    dplyr::filter(
+        data_split == "test",
+        shuffled == "False"
+    )
```

Comment:
Consider adding visualizations for F1 scores on training data and shuffled baseline on test data. I think the f1 scores on training data are particularly interesting to see how much different types of models overfit.

Reply:
My aim for the F1 score figure is to have a simple plot that includes a single number per phenotype, in fact, I might make it simpler!

The PR curves do the job of showing training vs. testing vs. shuffled baseline very well already :)